rabbitmq.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+
+  rabbitmq:
+    image: 'rabbitmq:3.6-management-alpine'
+    ports: 
+      # docker publish port 5672/15672 to 5672/15672 
+      # (將 docker 內部 ip 5672/15672, 跟外部 5672/15672 做連結)
+      - '5672:5672'
+      - '15672:15672'
+    environment:
+      RABBITMQ_DEFAULT_USER: "worker"
+      RABBITMQ_DEFAULT_PASS: "worker"
+      RABBITMQ_DEFAULT_VHOST: "/"
+    networks:
+      - dev
+
+  flower:
+    image: mher/flower:0.9.5
+    command: ["flower", "--broker=amqp://worker:worker@rabbitmq", "--port=5555"]
+    ports: 
+      # docker publish port 5555 to 5555 
+      # (將 docker 內部 ip 5555, 跟外部 5555 做連結)
+      - 5555:5555
+    depends_on:
+      - rabbitmq
+    networks:
+      - dev
+    platform: linux/amd64
```

Comment:
https://github.com/FinMind/FinMindBook/blob/master/DataEngineering/Chapter5/5.5/5.5.2/rabbitmq.yml

I noticed that the original file doesn't include the 'platform' argument in the Docker Compose file. I'm curious if you encountered any issues that led you to include this argument?

Reply:
🤔 might need some effort to recall it

Reply:
That's okay, I believe we can proceed with the merge initially. In case we discover it's unnecessary, we can always remove it later.